test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,17 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  it("returns Fizz when passed a multiple of 3", () => {});
+  expect(fizzBuzz(3)).toBe('Fizz')
 
-  test("returns Buzz when passed a multiple of 5", () => {});
+  it("returns Buzz when passed a multiple of 5", () => {});
+  expect(fizzBuzz(5)).toBe('Buzz')
 
-  test("returns FizzBuzz when passed a multiple 3 and 5", () => {});
+  it("returns FizzBuzz when passed a multiple 3 and 5", () => {});
+  expect(fizzBuzz(15)).toBe('FizzBuzz')
+  expect(fizzBuzz(30)).toBe('FizzBuzz')
 
-  test("returns the number when it isn't a multiple of 3 or 5", () => {});
+  it("returns the number when it isn't a multiple of 3 or 5", () => {});
+  expect(fizzBuzz(4)).toBe(4)
+  expect(fizzBuzz(11)).toBe(11)
```

Comment:
Don't max both single and double quotation marks in one project. In this project, eslint has been setup to use double quotation marks so should you.